examples/.gitignore

```diff
@@ -26,6 +26,7 @@ legacy/consumer_channel_example/consumer_channel_example
 legacy/producer_channel_example/producer_channel_example
 library-version/library-version
 mockcluster_example/mockcluster_example
+mockcluster_example/mockcluster_failure_example
```

Comment:
Just out of curiosity: why is this in .gitignore?

Reply:
HI. 
Right... both entries for the mock cluster examples seams to be unnecessary. Looks like a copy&paste error.